packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -55,23 +55,19 @@
   <script>
     harness.addEventListener('scenario-change', () => {
       const {scenario} = harness;
-
-      modelViewer.addEventListener('model-visibility', (event) => {
-        if (event.detail.visible) {
-          //TODO: this is a temporary workaround for taking screenshots before the model has been loaded
-          //This will be fixed when the screenshot function is refactored.
-          setTimeout(()=>{
-            self.dispatchEvent(new CustomEvent('model-ready'));
-          }, 500);
-        }
-      }, { once: true });
+      modelViewer.addEventListener('poster-dismissed', ()=>{
+        this.dispatchEvent(new CustomEvent('model-ready'));
+      }, {once: true});
       
```

Comment:
The poster-dismissed event happened after the model-visibility event, (i think the order is announceVisibility(fire model-visibility event) -> wait for transition finish -> fire poster-dismissed event) so i think it's ok to just keep only one event listener for poster-dismissed event.